models/ens/ens_reverse_latest.sql

```diff
@@ -16,37 +17,36 @@ with node_names as (
     select
     name,node,block_time,tx_hash
     from (
-        select
-        case when _name = '0x0000000000000000000000000000000000000000' then null else _name end as name
-        ,node
-        ,call_block_time as block_time
-        ,call_tx_hash as tx_hash
-        ,row_number() over (partition by node order by call_block_time desc) as ordering --in theory we should also order by tx_index here
+        select case when _name = '0x0000000000000000000000000000000000000000' then null else _name end as name
```

Comment:
are you sure this shouldn't remove the quotes, `_name` isn't varbinary?

Reply:
It seems `_name` column is varchar typed in the decoded table.
https://dune.com/queries/2756749?category=decoded_project&namespace=ethereumnameservice&contract=DefaultReverseResolver&blockchains=ethereum&abi=setName